pr-data.csv

```diff
@@ -3133,7 +3133,8 @@ https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.singleSslSocketFactory,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.singleVerifier,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.verifierAccepts,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
-https://github.com/kiegroup/optaplanner,dff7457cb5ba1557727deddef5b675dd00fbf8a8,optaplanner-core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,,,
+https://github.com/kiegroup/optaplanner,dff7457cb5ba1557727deddef5b675dd00fbf8a8,optaplanner-core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,MovedOrRenamed,,https://github.com/kiegroup/optaplanner/commit/4dd2dc8700a1972ba3148e2f3fabe5568f0b0c69
+https://github.com/kiegroup/optaplanner,4dd2dc8700a1972ba3148e2f3fabe5568f0b0c69,core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,,,
```

Comment:
Did you confirm that this was still ID in that commit `4dd2dc8700a1972ba3148e2f3fabe5568f0b0c69`? Did you try it in the latest commit?

Reply:
I removed the 2 rows of non-pr update for now, I'll work on that later. Now this pr only contains pr updates.